Multisig/UI/UI Library/Overlays/SnackbarView.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  SnackbarView.swift
+//  Multisig
+//
+//  Created by Dmitry Bespalov on 24.06.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+enum SnackbarViewMetrics {
+    static let textPadding = CGSize(width: 16, height: 14)
+    static let cornerRadius: CGFloat = 8
+    static let screenPadding = CGSize(width: 16, height: 10)
+    static let offscreenOffset: CGFloat = 300
+}
+
+struct SnackbarView<T: View>: View {
+
+    @Binding var isPresented: Bool
+    // this is to dynamically react to the bottom bar appearance
+    @Binding var bottomSpacing: CGFloat
+
+    var content: T
+
+    var body: some View {
+        ZStack(alignment: .bottom) {
+            Rectangle().opacity(0)
+
+            VStack {
+                self.content
+            }
+            .padding(.horizontal, SnackbarViewMetrics.textPadding.width)
+            .padding(.vertical, SnackbarViewMetrics.textPadding.height)
+            .frame(maxWidth: .infinity)
+            .foregroundColor(.gnoSnowwhite)
+            .background(Color.gnoDarkBlue)
+            .cornerRadius(SnackbarViewMetrics.cornerRadius)
+            .padding(.horizontal, SnackbarViewMetrics.screenPadding.width)
+            .gnoShadow()
+            .offset(y: self.isPresented ?
+                -self.bottomSpacing : SnackbarViewMetrics.offscreenOffset)
+        }
+        .opacity(isPresented ? 1 : 0)
+        .animation(.spring())
+    }
+
+}
+
+struct SnackbarView_Previews: PreviewProvider {
```

Comment:
So did you manage to fix preview? Mine is still not working.

Reply:
ah, no, it doesn't work